python/services/dataservice/dmod/dataservice/__main__.py

```diff
@@ -6,14 +6,131 @@
 )
 
 import argparse
-from . import name as package_name
-from .service import ServiceManager
-from .service_settings import ServiceSettings
-from .dataset_manager_collection import DatasetManagerCollection
-from dmod.scheduler.job import DefaultJobUtilFactory
 from pathlib import Path
 from socket import gethostname
 
+from dmod.scheduler.job import DefaultJobUtilFactory
+from dmod.modeldata.data.filesystem_manager import FilesystemDatasetManager
+from dmod.modeldata.data.object_store_manager import ObjectStoreDatasetManager
```

Comment:
I think I didn't do this originally because I wanted to reduce the number of modules in which these types got imported (though I don't remember _why_ I wanted to do that).  That probably isn't worth worrying about - it likely does more harm than good (if any) by cluttering what's in the _service.py_ file - but I'll mention it just as a sanity check.